linkerd/protocol/h2/src/main/scala/io/buoyant/linkerd/protocol/h2/H2AddForwardedHeaderConfig.scala

```diff
@@ -0,0 +1,68 @@
+package io.buoyant.linkerd.protocol.h2
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.{ServiceFactory, Stack, Stackable}
+import com.twitter.finagle.buoyant.h2.{Request, Response}
+import io.buoyant.router.http.{AddForwardedHeader, LabelerConfig}
+
+case class H2AddForwardedHeaderConfig(
```

Comment:
I think everything in the AddForwardedHeaderConfig and AddForwardedHeaderConfig.module is request/response type agnostic.  This means that we should be able to just have one of these (not separate ones for H1/H2) and it can live in http-base.